### ISSUE TITLE
CommandBar's flyout portion needs also to be rounded

### ISSUE THREAD

**Issue by chigy** [microsoft/microsoft-ui-xaml#1431]
CommandBar's flyout seen below should also get rounded corner.
![image](https://user-images.githubusercontent.com/22958680/66606767-a9890980-eb67-11e9-85ae-ee2e5046bae9.png)

Checked with design team and this should follow the flyout pattern of 4px all corners.

**Comment by robloo** [comment]
Are you sure this should be rounded? This is not a standard flyout and instead is an extension of the commands in the CommandBar which are still rectangular. You said it yourself:

> It is by design per "T-rule" when multiple items are next to each other and forms a T, it should not be rounded. Thus, we don't round list items nor grid items.

It's going to look visually strange to have a rounded flyout in this case -- but ultimately your design team makes the call of course.

Another comment here, the flyout now opens with a top placement mode. Was this changed from bottom intentionally? It's a little strange as the CommandBar is usually at the top of the app window and then the flyout appears sticking out above the app window. Bottom placement visually looked a lot better.

**Comment by mdtauk** [comment]
Outer edges should be rounded, but those edges that touch each other, would not be rounded.

**Comment by robloo** [comment]
@mdtauk Yes, I realize that. "edges that touch each other, would not be rounded" is what I was talking about. The commands in the flyout could be secondary or could just be overflowed primary commands. They are still grouped next to each other and logically 'touch'. I would argue this is a special case as the command bar, even with the flyout, is very much a continuous list of commands.

**Comment by chigy** [comment]
@mdtauk , @robloo , we reviewed this with design and we have concluded the right thing to do is to round all corners because there is no bar in this scenario. It is typically a transparent background with buttons on it, so it is very strange to have a corner that is not round. We also emphasized the separation by adding shadow a few releases ago. So the plan is to round all 4 corners. T-rule does not apply here since it does not intersect.
